### ISSUE TITLE
Modal body is overflowing with any size on Firefox

### ISSUE THREAD

**Issue by taroj1205** [yamada-ui/yamada-ui#3556]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.6.3

### Description

The modal body is overflowing with any size on Firefox.

![image](https://github.com/user-attachments/assets/9d8c5328-be89-4f86-91e1-3e12b4dbe418)

### Link to Reproduction

https://yamada-ui.com/components/overlay/modal

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

This is on Chrome.

![image](https://github.com/user-attachments/assets/d448a235-5c0f-4c44-8015-c552023817c9)

**Comment by hirotomoyamada** [comment]
@taroj1205 

Hi, I cannot reproduce this on Firefox on my Mac. Thanks for your time, but could you please investigate?

**Comment by taroj1205** [comment]
> @taroj1205
> 
> Hi, I cannot reproduce this on Firefox on my Mac. Thanks for your time, but could you please investigate?

Hi @hirotomoyamada, I have tried on a few firefox browsers and its forks.
Below is the list of the browsers I encountered the issue:
- Firefox (133.0)
- Zen Browser (133.0)
- Floorp (128.5.0)

Without issue:
- Firefox Developer Edition (134.0b3)
- Firefox Nightly (135.0a1)

It might be possible that it is fixed on later version of Firefox.
I would like someone else to confirm on Firefox v133.0 to see if issue occur on their Firefox too.
